core/src/main/java/com/linecorp/armeria/server/AbstractVirtualHostBuilder.java

```diff
@@ -221,6 +216,25 @@ public B serviceUnder(String pathPrefix, Service<? super HttpRequest, ? extends
     }
 
     /**
+     * Binds the specified {@link Service} at the the specified path pattern. e.g.
+     * <ul>
+     *   <li>{@code /login} (no path parameters)</li>
+     *   <li>{@code /users/{userId}} (curly-brace style)</li>
+     *   <li>{@code /list/:productType/by/:ordering} (colon style)</li>
+     *   <li>{@code exact:/foo/bar} (exact match)</li>
+     *   <li>{@code prefix:/files} (prefix match)</li>
+     *   <li><code>glob:/~&#42;/downloads/**</code> (glob pattern)</li>
```

Comment:
I had to use the `<code/>` tag to escape `*` in `/~*/` because `*/` is considered as the end of the comment block.